03_letters/mohankrishna/main.go

```diff
@@ -0,0 +1,37 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"sort"
+	"strings"
+)
+
+var out io.Writer = os.Stdout
+
+func main() {
+	fmt.Fprintln(out, strings.Join(sortLetters(letters("aba")), "\n"))
+}
+
+func letters(s string) map[rune]int {
+	letterMap := make(map[rune]int)
+	for _, char := range s {
+		letterMap[char]++
+	}
+	return letterMap
+}
+
+func sortLetters(m map[rune]int) []string {
+	keys := []int{}
+	for k := range m {
+		keys = append(keys, int(k))
+	}
+	sort.Ints(keys)
+
+	var sortedKeys = make([]string, len(keys))
```

Comment:
:+1: for pre-allocating the size

you can remove the `var` and use `:=` here, which follows the same style as the other declarations in this function.

Reply:
Fixed it!